### DISCUSSION TITLE
Screen is garbled on first installation

### DISCUSSION THREAD

**Discussion by stariontech** [elementary/triage#528] [Q&A]
First time user of elementary version 7.1.  Attempting installation on an Early 2009 Mac Mini.  Boot to the thumb drive with elementary, created with Etcher.  System appears to boot, but screen is all garbled, can't read anything.  Next steps?

**Comment by stariontech** [comment]
Re-downloaded using Windows, and flashed with Windows Balena.  No change.

As a side note, flashed LinuxMintCinnamon and was able to install it with no problems.  So an elementary.io installer issue?

**Comment by stariontech** [comment]
![Z2EuurVzQs2FrySsT2IsIA](https://github.com/elementary/triage/assets/12043819/0e0e0bd0-e10a-4462-b492-ddb51da4a4d6)

**Comment by vwtyler** [comment]
You could try `nomodeset` by booting to the terminal.

```
sudo nano /etc/default/grub

#Find the line starting with GRUB_CMDLINE_LINUX_DEFAULT and add nomodeset
GRUB_CMDLINE_LINUX_DEFAULT="quiet splash nomodeset"

#Save and exit 

#Update GRUB 
sudo update-grub

#Reboot
sudo reboot
```
